readme.md

```diff
@@ -1,60 +1,35 @@
 # internships
-
 Website for apprenticeship programme.
-
-## Usage & Development
-
-### 1. Create `.env` file based on `.env.example`:
-
+# Architecture
+The project contains a modified default configuration and is structured as follows:
+* `backend` stores configuration for Composer and Laravel application and other PHP related stuff;
+* `frontend` is for Node.js npm, Vite, Vue.js and lint configuration - it also contains assets used in the webpage;
+
+## Setup and Run
+### 1. Run setup script:
+The script will ask you to input a MapBox public key, which will be written into a .env file as VITE_MAPBOX_TOKEN:
 ```shell script
-cp .env.example .env
+bash setup.sh
```

Comment:
Can we use `sh`?

Reply:
Yes. 
It was bash before, because `sh` didn't understand double `[` in if statements and that's required in regex. It's gone now, so we can do that.

There's one inconsistency though - the setup file has `#!/bin/bash` at the top, but `sh` forces it to be `#!/bin/sh` if I understood correctly.

Should I update that line?

Reply:
Yes.